renku/cli/init.py

```diff
@@ -388,6 +389,30 @@ def init(
             )
             template_data = templates[template_num - 1]
 
+    # verify variables have been passed
+    template_variables = template_data.get('variables', {})
+    template_variables_keys = set(template_variables.keys())
+    input_parameters_keys = set(parameter.keys())
+    for key in (template_variables_keys - input_parameters_keys):
+        value = click.prompt(
+            text=(
+                f'The template requires a value for "{key}" '
+                f'({template_variables[key]})'
+            ),
+            default='',
+            show_default=False
+        )
+        parameter[key] = value
+    useless_variables = input_parameters_keys - template_variables_keys
+    if (len(useless_variables) > 0):
+        click.echo(
+            'These parameters are not required by the template: {}'.format(
```

Comment:
```suggestion
            INFO + 'These parameters are not used by the template and were ignored:\n\t {}'.format(
```

together with `from renku.core.commands.echo import INFO`

Reply:
oh nice! I didn't know about `INFO`. Thanks!